### PR TITLE
Auto assign issues to the project

### PR DESCRIPTION
**Pull request by VadymBezpalko**
## Description
Auto assigning new feature requests / bug reports to the DS project.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        3. Scroll down to '....'
+        4. See an error
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: A clear and concise description of what you expected to happen.
+    validations:
+      required: false
```
Comment (mreszke) [inline .github/ISSUE_TEMPLATE/bug_report.yml:36]: IMO it should be required.
I'm also missing the `actual behavior`

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      description: |
+        examples:
+          - **Design system package(s) version**: [e.g. @livechat/design-system-react-components@1.0.0-beta.11]
+          - **Device**: [e.g. iPhone6]
+          - **OS**: [e.g. macOS]
+          - **Browser**: [e.g. chrome, safari]
+          - **Version**: [e.g. 22]
+      render: markdown
+    validations:
+      required: false
```
Comment (mreszke) [inline .github/ISSUE_TEMPLATE/bug_report.yml:55]: IMO the package version is also important and should be required. Maybe it should be a separate field?

Comment (VadymBezpalko) [reply .github/ISSUE_TEMPLATE/bug_report.yml:55]: Good point. I'll add it separately.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (VadymBezpalko) [reply .github/ISSUE_TEMPLATE/bug_report.yml:36]: Ok, I will make it as required.
Regarding `actual behavior` - I thought that this kind of information would be included in the bug description and steps to reproduce sections.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    attributes:
+      label: Steps To Reproduce
+      description: Steps to reproduce the behavior.
+      placeholder: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See an error
+    validations:
+      required: false
```
Comment (mreszke) [inline .github/ISSUE_TEMPLATE/bug_report.yml:30]: would set that as required also
```suggestion
      required: true
```

**Review by mreszke — APPROVED** [review]

**Pull request merged**